src/styles/blocks/content.scss

```diff
@@ -0,0 +1,81 @@
+.clock {
+  position: relative;
+  width: 1em;
+  height: 1em;
+  box-sizing: border-box;
+  border: $padding-clock solid #000;
+  border-radius: 50%;
+  font-size: $size-clock;
+  text-align: center;
+
+  &__hand {
+    position: absolute;
+    margin-left: -0.02em;
+    margin-top: -0.02em;
+    height: 0.03em;
+    width: 0.03em;
+    left: 50%;
+    top: 50%;
+    background-color: #000;
+    border-radius: 50%;
+
+    &--time::after {
+      content: "";
+      border-radius: $border-radius;
+      position: absolute;
+      bottom: 50%;
+      left: 50%;
+      transform-origin: bottom;
+    }
+
+    &--minute::after {
+      background-color: #000;
+      margin-left: -0.01em;
+      height: 0.25em;
+      width: 0.03em;
+      animation: rotate-hand 3600s linear infinite;
+    }
+
+    &--second::after {
+      background-color: #f00;
+      margin-left: -0.01em;
+      height: 0.35em;
+      width: 0.02em;
+      animation: rotate-hand 60s linear infinite;
+    }
+  }
+
+  &__numbers {
+    padding: $padding-clock;
+    margin: 0;
+    height: 1em;
+    width: 1em;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+  }
+
+  &__number_vertical {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    position: absolute;
+    margin: auto -0.4em;
+    height: $size-clock - $padding-clock;
+    font-size: 0.2em;
+    top: 0%;
+    left: 50%;
+  }
+
+  &__number_horizon {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    position: absolute;
+    margin: -0.4em auto;
+    width: $size-clock - $padding-clock;
+    font-size: 0.2em;
+    top: 50%;
+    left: 0;
+  }
```

Comment:
you still duplicate a lot of styles here

Reply:
done